ci/build/build-packages.sh

```diff
@@ -43,14 +43,27 @@ release_gcp() {
   cp "./release-packages/$release_name.tar.gz" "./release-gcp/latest/$OS-$ARCH.tar.gz"
 }
 
+get_nfpm_arch() {
+  if jq -re ".${PKG_FORMAT}.${ARCH}" ./ci/build/arch-override.json > /dev/null; then
+    jq -re ".${PKG_FORMAT}.${ARCH}" ./ci/build/arch-override.json
+  else
+    echo "$ARCH"
+  fi
+}
+
```

Comment:
Maybe we could add a comment above explaining why we have to override the arch? Or why we have this function?

Reply:
Sure! I can do that - just give me a minute.

Reply:
Only request then is to add a comment or two explaining why we have to override the arch and  then we should approve and merge!

Reply:
Done!